### DISCUSSION TITLE
pip3 install paddleocr Error

### DISCUSSION THREAD

**Discussion by BlankConan** [PaddlePaddle/PaddleOCR#12511] [Q&A] [answered]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：macOS M1，Python 3.9.6
- 版本号/Version：14.1.2  Paddle：  PaddleOCR：2.7.0.3 问题相关组件/Related components：PyMuPDF
- 运行指令/Command Code：pip3 install paddleocr
- 完整报错/Complete Error Message：Failed building wheel for PyMuPDF

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):
yes

完整日志：
[install.log](https://github.com/PaddlePaddle/PaddleOCR/files/13807001/install.log)

**Comment by himanshudhami** [comment, marked as answer]
This will work:
**brew install swig** 
first and then try 
**pip install PyMuPDF==1.20.0** 
and it will work.
